templates/403.html

```diff
@@ -0,0 +1,6 @@
+<h1>403: Permission Denied!</h1>
+<p>
+    You were denied permission to view this page because:
+    <br/>
+    {{exception}}
```

Comment:
Remember to not leak to much information about the exception because it can be used for an attack

Reply:
Yeah, I was worried about that too. But, in this case {{exception}} is actually just the exception message, as defined in the view throwing the error. 
I'll add a comment to explain this cause I agree it looks scary